mod/wikirate_project/set/abstract/info_pages.rb

```diff
@@ -0,0 +1,6 @@
+format :html do
+
+  def section_header blurb, button={}
```

Comment:
I ended up tweaking this on wikirate. It looks like you're not ever calling `section_header` with a block, so you could safely get rid of the yielded stuff if you wanted to.  (I originally added that because of complexity in the variety of section headers)